src/vcpkg/vcpkgpaths.cpp

```diff
@@ -776,6 +776,7 @@ If you wish to silence this error and use classic mode, you can:
         auto tar_cmd_builder = git_cmd_builder(*this, dot_git_dir, dot_git_dir)
                                    .string_arg("archive")
                                    .string_arg(git_tree)
+                                   .string_arg("--worktree-attributes")
```

Comment:
```suggestion
                                   .string_arg("-c").string_arg("core.autocrlf=false")
```
I think it may be better to instead directly set the relevant attribute instead of trying to incidentally find it in a `.gitattributes` file.

Reply:
Yes, that makes total sense. I've applied your suggestion.